app/controllers/api/v1/profiles_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Api::V1::ProfilesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    claimed_tasks = current_user.accepted_tasks.empty? ? "You don't have any claimed tasks." : current_user.accepted_tasks
+    created_tasks = current_user.tasks.empty? ? "You don't have any ongoing tasks." : current_user.tasks
```

Comment:
You don't have any open requests (line 8) assuming these were created by the user in question. Claimed tasks I'm assuming are for the same user, correct?

Reply:
We're sending two keys: Claimed and Created tasks. Inside Claimed tasks key you can see all the active tasks you claimed, if you didn't claim any it says the message that you don't have any claimed tasks. Inside Created tasks key it shows if you have any tasks you created to get help with if there isn't any you get the other message.

Reply:
So on a FE under User's profile you could have two buttons Created and Claimed where the user can view active tasks either created or confirmed

Reply:
Thanks for the explanation. Looks like we could now have a dashboard in the front end for each user.